pkg/networkservice/connectioncontext/ipcontext/ipaddress/common.go

```diff
@@ -42,7 +42,22 @@ func addDel(ctx context.Context, conn *networkservice.Connection, vppConn api.Co
 	if ipNets == nil {
 		return nil
 	}
+
+	curIPs, ok := load(ctx, isClient)
 	for _, ipNet := range ipNets {
+		// Сheck if the interface already has ipNet
```

Comment:
What if for some reason address is already assigned to the interface, but not during the Request with the same Connection.Id? In such case there will be no record about this in the metadata.
Probably we can get information about the interface IP addresses directly from VPP?